server/server.go

```diff
@@ -143,13 +144,33 @@ func (s *Server) Context() context.Context {
 	return s.ctx
 }
 
+// parseInvocation parses the start command in the same way we already do in the entrypoint
+// We can use this to set the container command with all variables replaced.
+func parseInvocation(invocation string, envvars map[string]interface{}, memory int64, port int, ip string) (parsed string) {
+	// replace "{{" and "}}" with "${" and "}" respectively
+	invocation = strings.Replace(invocation, "{{", "${", -1)
```

Comment:
Any reason not to use strings.ReplaceAll?  Seems equivalent and slightly more readable.

Reply:
because I was focused on each variable showing up once instead of thinking that it may show up more than once in the start command...